test/integration/targets/setup_tls/tasks/trusty.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: add python3.6 PPA
```

Comment:
This may impact tests that run after this one, need to check with @mattclay if this is something we want to do

You can do `skip/python2` so this test only runs on Python 3 https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/integration-aliases.html#skipping

Reply:
Fixed in: https://github.com/ansible/ansible/blob/dcdf636c6b036a2fe273a6eba12f44680312183d/test/integration/targets/setup_tls/tasks/cleanup.yml#L17

The TLS certificate generator requires Python 3.5 or higher to be installed, and the Trusty machine by default comes with Python 3.4. I would have picked another solution, but this works the best for RabbitMQ and is potentially quite reusable with other modules too (hence the generic name 'setup_tls')